### DISCUSSION TITLE
page names

### DISCUSSION THREAD

**Discussion by iichernenko** [zim-desktop-wiki/zim-desktop-wiki#2255] [Q&A]
H! 

How to include the symbol underline <_> in the page name?

**Comment by introt** [comment]
Hi! As far as I know, this is currently not possible due to `_` being used to represent spaces in filenames. This limitation is due to the way Zim stores pages as text files on the file system. You can, however, use `_` in the page title, eg. the top-level heading.

**Reply by introt** [reply]
Other characters not allowed in page names include `:` and `+` (Zim link syntax), `/` and `\` (path separators on 
*nix and Windows), as well as `?`, `#`, `*`, `'`, `<`, `>`, `|`, `%`, tabs and newlines.
